### PR TITLE
Melbourne Sunbury Corridor & standardisation

### PR DESCRIPTION
**Pull request by mattkelly4**
Added more notes for TWR controllers to handle Sunbury Corridor and standardised presentation of standard assignable headings.

### CODE REVIEW

File: docs/aerodromes/Melbourne.md [hunk 1]
```diff
@@ -14,17 +14,31 @@
...
+Due to the close proximity of the airspace to the arrival and departure paths at YMML, controllers should be aware of surrounding traffic before issuing a clearance to helicopters.  If necessary, issue a clearance limit for separation, then instruct helicopters to report sighting and maintain own separation with other aircraft.  Alternatively, tower controllers can simulate visual separation provided no risk of collision exists and both aircraft remain in sight of the controller at all times.
+
+!!! example
+    **HEMS3:** "Melbourne Tower, HEMS3, approaching Sunbury Water Tank, A020, for Essendon via the Sunbury Corridor, request clearance"  
+    **ML ADC:** "HEMS3, cleared to Essendon, track Sunbury Corridor, not above A020, clearance limit Powerline Crossing"  
+    **HEMS3:** "Cleared to Essendon, track Sunbury Corridor, not above A020, clearance limit Powerline Crossing, HEMS3"  
+
+    **ML ADC:** "HEMS3, report sighting a Jetstar A320, 4nm final runway 16"  
+    **HEMS3:** "Traffic sighted, HEMS3"  
+    **ML ADC:** "HEMS3, cleared direct Melbourne, thence Essendon, pass behind that aircraft and maintain own separation"
```
Comment (alexd-332) [inline docs/aerodromes/Melbourne.md:34]: Perhaps a little bit pedantic, but technically should be an additional check in there of "HEMS3, confirm you are able to maintain own separation with the Jetstar A320?"

Comment (alexd-332) [inline docs/aerodromes/Melbourne.md:34]: Also a bit pedantic, but would be good to standardise clearance phraseology across the board, ie "Cleared to YMEN via ML, maintain A020"

Comment (LeGiraffe) [reply docs/aerodromes/Melbourne.md:34]: Does there also need to be a "cancel clearance limit"?

**Review by tylerthetiletiler — DISMISSED** [review]

**Review by SamKeast — APPROVED** [review]

**Review by SamKeast — APPROVED** [review]

File: docs/aerodromes/Melbourne.md (hunk 1 above)
Comment (alexd-332) [reply docs/aerodromes/Melbourne.md:34]: You'd know better than me

Comment (LeGiraffe) [reply docs/aerodromes/Melbourne.md:34]: 9.2.2.12 Cancelling clearance limit
When a clearance limit is cancelled, issue an onwards clearance specifying the level
and route to be flown from that point.

If we want to teach Clearance Limits, it'll need a bit more then...
"HEMS3, Cancel Clearance Limit, Pass Behind A320 4NM Final Runway 16, thence Cleared Melbourne, Essendon, 2000ft, Caution Wake Turbulence."

Comment (mattkelly4) [reply docs/aerodromes/Melbourne.md:34]: Maybe I'll remove the clearance limit and just give the pass behind instruction, for simplicity in the example, and we can address clearance limits in controller skills separately?

Comment (LeGiraffe) [reply docs/aerodromes/Melbourne.md:34]: Concur

**Review by alexd-332 — APPROVED** [review]

**Review by SamKeast — APPROVED** [review]

**Review by tylerthetiletiler — APPROVED** [review]

**Pull request merged**